jekyll/_cci2/hello-world-macos.md

```diff
@@ -56,16 +59,25 @@ to indicate what is happening at each step.
 
 ```yaml
 version: 2.1
+
 jobs: # a basic unit of work in a run
-  build: # runs not using `Workflows` must have a `build` job as entry point
-    macos:  # indicate that we are using the macOS executor
+  test: # your job name
+    macos:
       xcode: 11.3.0 # indicate our selected version of Xcode
     steps: # a series of commands to run
       - checkout  # pull down code from your version control system.
       - run:
-          # run our tests using xcode's cli tool `xcodebuild`
           name: Run Unit Tests
           command: xcodebuild test -scheme circleci-demo-macos
+      - run:
```

Comment:
Have you tested this?
I feel as though you run `Build Application` twice?
If you wanted to scope to only run tests, then it should do that.